### ISSUE TITLE
__init__() missing 1 required positional argument: 'hidden_dim'

### ISSUE THREAD

**Issue by clearly-outsane** [Benjamin-Etheredge/mlp-mixer-keras#2]
![image](https://user-images.githubusercontent.com/44093134/119375451-7366b200-bcd8-11eb-8a3e-d2f33752f388.png)

I'm just trying to load the model that I saved, any idea why I'm getting this? I even implemented a custom get_config function but it still throws the same error. The model trains and saves fine but I just cannot load it back

**Comment by Benjamin-Etheredge** [comment]
I suspected there could be issues with config loading. I'll take a look.

**Comment by clearly-outsane** [comment]
Oh I just fixed this, yeah there was a problem with my get_config() function, I'll comment what I did later :3

**Comment by Benjamin-Etheredge** [comment]
Happy to hear you got it working! 
I managed to reproduce the issue only when passing those custom objects to `load_model`. Was that your issue? I would still call it a bug, so I pushed a patch that should fix it anyway (`mlp-mixer-keras==0.05`).

**Comment by clearly-outsane** [comment]
Yep that was my issue. I just checked, that was exactly what I did too - just returned a config. Thanks for the quick response! 

Edit - You haven't assigned activation in the config.update for MlpBlock - was that intentional?
